pipeline/tests/powdr_std.rs

```diff
@@ -115,6 +115,38 @@ fn permutation_via_challenges_ext() {
         .unwrap();
 }
 
+#[test]
+fn lookup_via_challenges_bn() {
+    let f = "std/lookup_via_challenges.asm";
+    test_halo2(f, Default::default());
+}
+
+#[test]
+fn lookup_via_challenges_ext() {
+    let f = "std/lookup_via_challenges_ext.asm";
+    test_halo2(f, Default::default());
+    // Note that this does not actually run the second-phase witness generation, because no
+    // Goldilocks backend support challenges yet. But at least it tests that the panic from
+    // the previous test is not happening.
```

Comment:
I think the "previous test" reference doesn't apply here, also in the next test.